File: tests/fixtures/dap/hello.py
def greet(name):
    message = f"Hello, {name}!"
    return message


def main():
    x = 42
    y = x + 1
    result = greet("world")
    print(f"x={x} y={y}")
    print(result)
    return 0


if __name__ == "__main__":
    main()

File: tests/fixtures/java/Hello.java
public class Hello {
    public static void greet(String name) {
        String message = "Hello, " + name + "!";
        System.out.println(message);
    }

    public static void main(String[] args) {
        int x = 42;
        int y = x + 1;
        greet("World");
        System.out.println("x=" + x + " y=" + y);
    }
}
